Tests/BreApi.cs

```diff
@@ -0,0 +1,34 @@
+using NUnit.Framework;
+using SeaPublicWebsite.ExternalServices.Models;
+using FluentAssertions;
+
+namespace Tests;
+
+public class BreApi
+{
+    [SetUp]
+    public void Setup()
+    {
+    }
+
+    [Test]
+    public void RequestsMeasures()
+    {
+        BreRequest request = new(
```

Comment:
Please add `Arrange`, `Act`, `Assert` comments to tests so it's clear what's going on where.
In this case `Arrange` and `Act` are the same step (which is fine)